go.mod

```diff
@@ -91,6 +91,7 @@ replace (
 	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.24.2
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.24.2
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.13.0
+	vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
```

Comment:
also, is it necessary to include these changes from go.mod and go.sum?

Reply:
This is required on my local to successfully execute `go mod tidy`. I'm not sure if it affects tests in general, do you want me to drop it?

Reply:
I think it's something specific to Fedora only, if I'm not wrong. We can include it, but I hope it doesn't break things for non-Fedora users.

Reply:
I can confirm that with this change, running tests on Mac works fine.